org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/tasks/TaskMarkerContributor.java

```diff
@@ -65,8 +66,8 @@ protected void createTaskMarkers(IFile file, List<Task> tasks, IProgressMonitor
 	public void deleteMarkers(IFile file, IProgressMonitor monitor) {
 		try {
 			file.deleteMarkers(TaskMarkerTypeProvider.XTEXT_TASK_TYPE, true, IResource.DEPTH_ZERO);
-		} catch (Throwable e) {
-			throw new RuntimeException(e);
+		} catch (CoreException e) {
+			Exceptions.sneakyThrow(e);
```

Comment:
excedption handling is still bogus in 
org.eclipse.xtext.builder.builderState.MarkerUpdaterImpl.processDelta(Delta, ResourceSet, IProgressMonitor)
case where delete markers is dirctly called. wonder if we should catch and log there too

Reply:
wonder how to run in the delta.getNew() == case

Reply:
https://ci.eclipse.org/xtext/job/xtext/job/cd_restore_deleteMarkers_b/1/ 
seems to be ran by some tests

Reply:
<img width="874" alt="Bildschirmfoto 2023-07-11 um 17 19 25" src="https://github.com/eclipse/xtext/assets/238736/f9d6db69-7b69-4309-89c8-ab2338d0bb7e">
exception will fall though, be caught and rethrown here
and finally handled here
<img width="1305" alt="Bildschirmfoto 2023-07-11 um 17 21 04" src="https://github.com/eclipse/xtext/assets/238736/a3d7bde7-b8ed-4623-b40f-50b21cf1f3cb">
this makes limited sense to me. @szarnekow what do you think?

Reply:
other code like
org.eclipse.xtext.ui.validation.MarkerEraser.deleteValidationMarkers(IFile, CheckMode, IProgressMonitor)
org.eclipse.xtext.ui.validation.MarkerEraser.deleteValidationMarkers(IFile, CheckMode, IProgressMonitor)
directly catch and log
=> propose to change to

```
@Override
	public void deleteMarkers(IFile file, IProgressMonitor monitor) {
		try {
			file.deleteMarkers(TaskMarkerTypeProvider.XTEXT_TASK_TYPE, true, IResource.DEPTH_ZERO);
		} catch (CoreException e) {
			LOG.error(e.getMessage(), e);
		}
	}
	```

Reply:
I agree. We shouldn't fail miserably because some marker wasn't created

Reply:
thx. updated the pr